### ISSUE TITLE
Sending 50k from Breez Mobile to SDK failed with `no_route`

### ISSUE THREAD

**Issue by ok300** [breez/breez-sdk-greenlight#809]
I tried to send 50k sats from Breez Mobile to the Pulse node. It failed several times with `no_route`.

Other tests I did for the same sender / receiver:
- sent 5k successfully
- sent 40k successfully
- (50k still failed)

Other tests I did for the same sender, *different* GL recipient:
- sent 50k successfully
- received 50k successfully (GL -> Breez Mobile)

Other tests I did for the same recipient (Pulse node) :
- received 50k successfully from Phoenix (same 50k invoice that failed above)

On the sender side (Breez Mobile), I also tried `resetmc`, Update Graph, and restarting the app, with the 50k invoice still failing to pay.

So in summary
- it's not an issue with the invoice (as it could be paid from Phoenix, and it had the correct hint)
- it's not an issue with the Breez Mobile sender (as it could pay 50k invoices to another GL node)
- it's not an issue with the receiver (as it could receive 50k from another sender)

One "special" thing about the receiver Pulse node is that it has a huge history (~18k payments), but that was completely synchronized when payments were attempted.
